aptos-move/aptos-release-builder/src/components/framework.rs

```diff
@@ -74,16 +74,15 @@ pub fn generate_upgrade_proposals(
         // 1-aptos-stdlib.move	3-aptos-token.move	5-version.move		7-consensus-config.move
         // The first framework file being generated is 3-aptos-token.move. It's using the next_execution_hash being passed in (so in this case, the hash of 4-gas-schedule.move being passed in mod.rs).
         // The second framework file being generated would be 2-aptos-framework.move, and it's using the hash of 3-aptos-token.move (which would be result.last()).
-        let mut _execution_hash: String = "".to_owned();
-        if !next_execution_hash.is_empty() {
+        let mut _next_execution_hash_string = "".to_owned();
+        if !next_execution_hash.clone().is_empty() {
             args.push("--next-execution-hash");
             if result.is_empty() {
-                args.push(&next_execution_hash);
+                _next_execution_hash_string = hex::encode(next_execution_hash.clone());
             } else {
-                _execution_hash =
-                    HashValue::sha3_256_of(result.last().unwrap().1.as_bytes()).to_string();
-                args.push(&_execution_hash);
+                _next_execution_hash_string = hex::encode(get_execution_hash(&result));
```

Comment:
Why do we need to do hex encode here?

Reply:
need to convert from bytes to string for the CLI command